### DISCUSSION TITLE
Has anyone had any luck getting SettingsCard from Communitytoolkit 8 to work on anything other than windows

### DISCUSSION THREAD

**Discussion by cconner100** [unoplatform/uno#15904] [General]
I thought this was a x-platform system :)

Here is a sample application on this simple control, shows just fine in Windows but causes an exception on anything else.

I also put in a bug request on the communitytoolkit windows forum but no replies

[TestSettingsCard.zip](https://github.com/unoplatform/uno/files/14631629/TestSettingsCard.zip)

**Comment by HaoLife** [comment]
Recompile the source code and reference the new package

**Reply by cconner100** [reply]
OK i did and get the same error,

here is the list and version of nugets i am working with
```
<PackageVersion Include="CommunityToolkit.Mvvm" Version="8.2.2" />
    <PackageVersion Include="CommunityToolkit.WinUI.Animations" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Extensions " Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Helpers " Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Triggers " Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.SettingsControls" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Segmented" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Primitives" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Collections" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.Sizers" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Lottie" Version="8.0.0-rc" />
    <PackageVersion Include="CommunityToolkit.WinUI.Behaviors" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.DeveloperTools" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Media" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.HeaderedControls" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.LayoutTransformControl" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.TokenizingTextBox" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.RangeSelector" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.RadialGauge" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.CameraPreview" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.RichSuggestBox" Version="8.0.240109" />
    <PackageVersion Include="CommunityToolkit.WinUI.Controls.MetadataControl" Version="8.0.240109" />
```

Were you able to get my sample project to display the settingscard on android?

**Reply by jeromelaban** [reply]
Can you provide which exception you're getting? 

Also note that community toolkit 8 does not have a stable version available for Uno 5+. Preview builds are available in their [NuGet development feed](https://pkgs.dev.azure.com/dotnet/CommunityToolkit/_packaging/CommunityToolkit-MainLatest/nuget/v3/index.json), if you want to try it out.

**Reply by cconner100** [reply]
this is the exception on frame load
Method not found: void Microsoft.UI.Xaml.FrameworkElement.remove_IsEnabledChanged(Microsoft.UI.Xaml.DependencyPropertyChangedEventHandler)

here is the stack
Frame.InnerNavigate() at /Users/curtisconner/Library/Application Support/JetBrains/Rider2024.1/resharper-host/SourcesCache/932bbfbe497839bdd3150a9c3ca5a6bbbcb528ddb6f35e93a17d930b4728942/Frame.cs:line 309
[External code: 2 frames]
App.OnLaunched()
AppHead.OnLaunched()
[External code: 4 frames]
EntryPoint.Main()

**Comment by HaoLife** [comment]
https://github.com/HaoLife/CircleApp/tree/main/CommunityToolkit.WinUI.Controls.SettingsControls

Copy out from this project

**Reply by cconner100** [reply]
any update on this issue? can we expect a fix anytime soon?

**Reply by jeromelaban** [reply]
You can follow the progress on the community toolkit's repo: https://github.com/CommunityToolkit/Windows/discussions/383

**Reply by agneszitte** [reply]
@cconner100, @HaoLife you will have some details regarding SettingsCard here: https://github.com/unoplatform/uno/issues/16073

Our documentation regarding Windows Community Toolkit will be updated soon.
Here is the related issue with all the details: https://github.com/unoplatform/uno/issues/16872
